doc/chapters/gateways/index.rst

```diff
@@ -422,7 +422,16 @@ Wayland gateway
 
 The Wayland Gateway is used to provide access to the hsot system Wayland server.
 
+Please notice that the Wayland Gateway only provides access to the Wayland
+socket and sets the ``XDG_RUNTIME_DIR`` accordinly. For a Wayland
+*capability*, it is also necessary to ensure that the contained application
+has access to the graphics hardware devices and associated libraries.
```

Comment:
Actually, access to hardware devices does not seem to be necessary, at least not when testing. We are using the wayland gateway and running "weston-info" inside the container, which works. No device nodes are copied into the container. Perhaps the test is too rudimentary, I don't know.